package.json

```diff
@@ -17,8 +17,11 @@
     "start": "node server",
     "start-dev": "NODE_ENV='development' npm run build-client-watch & npm run start-server",
     "start-server": "nodemon server -e html,js,scss --ignore public --ignore client",
-    "test": "NODE_ENV='test' mocha ./server/**/*.spec.js ./server/**/**/*.spec.js ./client/**/*.spec.js --require @babel/polyfill --require @babel/register",
-    "test-watch": "NODE_ENV='test' mocha -w ./server/**/*.spec.js ./server/**/**/*.spec.js ./client/**/*.spec.js --require @babel/polyfill --require @babel/register"
+    "test-client": "karma start",
+    "test-client-watch": "karma start --no-single-run",
```

Comment:
Test running is split between `client` and `server` tests.

`npm run test` will run both back-to-back.